src/rdkafka_int.h

```diff
@@ -397,55 +400,32 @@ struct rd_kafka_s {
                  *   Only one transactional API call is allowed at any time.
                  *   Protected by the rk_lock. */
                 struct {
-                        char name[64];        /**< API name, e.g.,
-                                               *   SendOffsetsToTransaction */
-                        rd_kafka_timer_t tmr; /**< Timeout timer, the timeout
-                                               * is specified by the app. */
-
-                        int flags; /**< Flags */
-#define RD_KAFKA_TXN_CURR_API_F_ABORT_ON_TIMEOUT                               \
-        0x1 /**< Set state to abortable                                        \
-             *   error on timeout,                                             \
-             *   i.e., fail the txn,                                           \
-             *   and set txn_requires_abort                                    \
-             *   on the returned error.                                        \
-             */
-#define RD_KAFKA_TXN_CURR_API_F_RETRIABLE_ON_TIMEOUT                           \
-        0x2 /**< Set retriable flag                                            \
-             *   on the error                                                  \
-             *   on timeout. */
-#define RD_KAFKA_TXN_CURR_API_F_FOR_REUSE                                      \
-        0x4 /**< Do not reset the                                              \
-             *   current API when it                                           \
-             *   completes successfully                                        \
-             *   Instead keep it alive                                         \
-             *   and allow reuse with                                          \
-             *   .._F_REUSE, blocking                                          \
-             *   any non-F_REUSE                                               \
-             *   curr API calls. */
-#define RD_KAFKA_TXN_CURR_API_F_REUSE                                          \
-        0x8 /**< Reuse/continue with                                           \
-             *   current API state.                                            \
-             *   This is used for                                              \
-             *   multi-stage APIs,                                             \
-             *   such as txn commit. */
+                        char name[64];     /**< API name, e.g.,
```

Comment:
the comments are a bit misaligned with others. not sure if this is intended. (by 6 spaces)

Reply:
Yeah it's a bit random, but we rely on clang-format to do formatting and the pros outweighs the cons :)